tools/isobuild/package-source.js

```diff
@@ -213,14 +213,44 @@ var getExcerptFromReadme = function (text) {
 class SymlinkLoopChecker {
```

Comment:
I wonder if we might be able to get rid of the `SymlinkLoopChecker` by just limiting the maximum depth of recursion to something "ridiculous" like 100? Symlink loops are a relatively rare occurrence.

Reply:
It seems to work correctly, but it does increase the size of the client bundle since it uses the most deeply nested version of the file it finds. You can see the app.js contents at https://gist.github.com/zodern/68a9cd266c42d7e1719dda4114d94b64

Instead of limiting the depth we could listen for the ELOOP error. Node throws that error once it gets into a depth of ~30 in a symlink loop